sensum-pwa/src/app/App.res

```diff
@@ -44,89 +44,17 @@ let make = () => {
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
-    <div> <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1> </div>
+  let items =
+    Belt.Array.rangeBy(0, 42, ~step=1)
+    ->Belt.Array.map(i => {
+      <div key={i->Belt.Int.toString}>
+        <h1 className="text-4xl text-purple-50"> {"element"->React.string} </h1>
+      </div>
+    })
+    ->React.array
```

Comment:
this is necessary in order to avoid:
```
This has type: array<React.element>
Somewhere wanted: React.element
```
^ similar scenario as rescript string vs react string 

(tip: is just the natural transformation `React.array`!)